main.js

```
@@ -43,6 +43,26 @@ window.addEventListener('load', (event) => {
         drawPile[j] = temp;
     }
 
+    // determines if 3 selected cards make a Set
+    function isSet(cardArray){
```

before the for loop, you probably should throw an error if the length of cardArray is not 3